egs/wsj/s5/steps/nnet3/cldnn/train.sh

```diff
@@ -0,0 +1,721 @@
+#!/bin/bash
+
```

Comment:
@danpovey there is redundancy in steps/nnet3/*/train.sh scripts as we add more architectures. Some of these scripts differ in nothing but the config generators.
e.g. cldnn/train.sh and lstm/train.sh would be very similar. cnn/train.sh tdnn/train.sh would be very similar.
 I think we should look at a better way to pass the nnet3 architecture parameters. I am thinking about passing all the make_config.py parameters using a config file so that steps/nnet3/*/train.sh script can just invoke the config generator as
<code>
steps/nnet3/$model/make_configs.py $config_file
</code>

Any suggestions ?